src/modules/expense/PriceSection.tsx

```diff
@@ -0,0 +1,67 @@
+import { styled, Theme } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import FormControl from '@mui/material/FormControl';
+import InputAdornment from '@mui/material/InputAdornment';
+import Input from '@mui/material/Input';
+import { colors } from 'src/styles/colors';
+import React from 'react';
+
+const StyledDiv = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flex: 'end',
+  flexDirection: 'column',
+  margin: '80px 30px 0 30px',
```

Comment:
Use spacing value from theme for all values defined in pixels

Reply:
spacing is not defined in theme.tsx, I haven't change anything in theme.tsx after cloned the project. So it uses default material ui spacing, should still change it? And how to change? margin: 'theme.spacing(10) theme.spacing(4) theme.spacing(0) theme.spacing(4)', like this?

Reply:
Yes, that's exactly how you do it :) `theme.spacing(4)` would be 32px. That's fine if you use a close value e.g. 32px instead of 30px. 
https://mui.com/material-ui/customization/spacing/